src/fragments/lib/storage/flutter/move.mdx

```diff
@@ -1,29 +1,29 @@
-import flutter0 from "/src/fragments/lib/storage/flutter/dev-preview-api-callout.mdx";
-
-<Fragments fragments={{flutter: flutter0}} />
-
-You can move an existing file to a different folder location in your S3 bucket. This requires the user who initiates the move operation to have read and write permission on the source file.
+You can move an existing file to a different folder location in your S3 bucket. The user initiating the move operation must have read and write permission on the source file.
```

Comment:
And the destination file?

Reply:
Ah yea, if the destination has existed, it needs write permission as well. Updated the doc